modules/ROOT/pages/policy-mule3-simple-security-manager.adoc

```diff
@@ -0,0 +1,33 @@
+= Simple Security Manager Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Simple Security Manager policy lets you configure a security manager with a username and password that you can define at the policy level.
+
+== Prerequisites
+
+This document assumes that you are an API Versions Owner for the API version that you want to manage, or that you are a member of the Organization Administrators role.
+
+== How Does This Policy Work?
+
+This policy acts as a simple security manager for you to test validations against your APIs using a username and password that you define when configuring your policy.
```

Comment:
What about something like this? 

This policy acts as a simple security manager to be used by policies which require one (such as Basic Authentication policy). It validates only **one** username and password combination. Usually used for testing/demonstrations purposes.